azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/service/sdk/CallingSDKWrapper.kt

```diff
@@ -56,6 +62,7 @@ internal class CallingSDKWrapper(
 
     private var videoDevicesUpdatedListener: VideoDevicesUpdatedListener? = null
     private var camerasCountStateFlow = MutableStateFlow(0)
+    private var lobby: Lobby? = null
```

Comment:
Do we need to keep a lobby object here? It can always be accessed from nullableCall?.lobby
This way we will not have a chance to leak this object, as now it is not cleaned.

Reply:
resolved